legion/engine/physics/colliders/convexcollider.cpp

```diff
@@ -797,7 +816,12 @@ namespace legion::physics
 
                 j++;
 
-                if (j > 999) { return; }
+                if (j > 999)
+                {
+                    edges.clear();
```

Comment:
This seems dangrous, maybe we want to do something with the gathered edges, even if they are not complete